config/networking.yml

```diff
@@ -14,9 +14,9 @@ cfroutesync:
   image: #@ data.values.images.cfroutesync
 
   ccCA: #@ base64.decode(data.values.system_certificate.ca)
-  ccBaseURL: #@ "https://api.{}".format(data.values.system_domain)
+  ccBaseURL: http://capi.cf-system.svc.cluster.local
```

Comment:
should change this to #@ "http://capi." + data.values.system_namespace + ".svc.cluster.local"

Reply:
Maybe we'd add util library for k8s stuff and in this case url resolve by service name and namespace